### ISSUE TITLE
Use `icmd` to test interactive SQL REPL

### ISSUE THREAD

**Issue by se7entyse7en** [src-d/engine#412]
Current the test of interactive SQL REPL doesn't use `icmd`, but it uses `os/exec` directly.

Context [here](https://github.com/src-d/engine/pull/398#issuecomment-480360020).

**Comment by smacker** [comment]
I don't see any problem with testing repl without `icmd`. This test is clearly different from others and I'm not sure such use case will be ever supported by `icmd` project.

**Comment by se7entyse7en** [comment]
> This test is clearly different from others

True, I just think that in case we're able to use `icmd` even in this case we have more consistency as we would have a single api for running commands. I haven't tried patching it locally, but it should be easy to try it and eventually open a PR to `icmd`.
